cmd/print.go

```diff
@@ -85,19 +86,24 @@ func (pr *PrintRunner) PrintGatewayAPIObjects(cmd *cobra.Command, _ []string) er
 		return fmt.Errorf("failed to initialize namespace filter: %w", err)
 	}
 
-	gatewayResources, err := i2gw.ToGatewayAPIResources(cmd.Context(), pr.namespaceFilter, pr.inputFile, pr.providers, pr.getProviderSpecificFlags())
+	gatewayResources, tables, err := i2gw.ToGatewayAPIResources(cmd.Context(), pr.namespaceFilter, pr.inputFile, pr.providers, pr.getProviderSpecificFlags())
 	if err != nil {
 		return err
 	}
 
-	pr.outputResult(gatewayResources)
+	pr.outputResult(gatewayResources, tables)
 
 	return nil
 }
 
-func (pr *PrintRunner) outputResult(gatewayResources []i2gw.GatewayResources) {
+func (pr *PrintRunner) outputResult(gatewayResources []i2gw.GatewayResources, tables []table.Writer) {
```

Comment:
I dont think tables are part of the result. The result are only the resulted resources.

So why not simply print it above line 94?
if you apply the comment I left below so you `tables` will be a list of already print-ready tables and you can just print it there (above line 94)

Reply:
Ok i will change this